crates/starknet-devnet-core/src/account.rs

```diff
@@ -104,14 +105,20 @@ impl Account {
     }
 
     fn eth_balance_storage_key(&self) -> DevnetResult<ContractStorageKey> {
-        let storage_var_address =
-            get_storage_var_address("ERC20_balances", &[Felt::from(self.account_address)])?;
+        let storage_var_address = starknet_types::patricia_key::PatriciaKey::new(Felt::new(
+            get_storage_var_address("ERC20_balances", &[FieldElement::from(self.account_address)])
+                .map_err(|_| Error::ProgramError)?
+                .to_bytes_be(),
+        )?)?;
```

Comment:
Wouldn't it be better to implement `TryFrom` for this and similar conversions, especially if the same conversion is done multiple times? Is it possible to do so in e.g. `patricia_key.rs` or `contract_storage_key.rs` or `felt.rs`?

Or would that be too similar to the original code with our custom `get_storage_var_address`? @marioiordanov

Reply:
it wouldn't be similar

Reply:
Talking about FieldElement -> PatriciaKey

Reply:
ok, i will work on that

Reply:
Done, please check.